webdriver/webapp_server.go

```diff
@@ -311,10 +312,83 @@ func addStaticData(i DevAppServerInstance) (err error) {
 	log.Println("Adding static TestRun data...")
 	for i := range staticTestRuns {
 		key := datastore.NewIncompleteKey(ctx, "TestRun", nil)
-		if _, err := datastore.Put(ctx, key, &staticTestRuns[i]); err != nil {
+		if key, err = datastore.Put(ctx, key, &staticTestRuns[i]); err != nil {
 			return err
 		}
+		staticTestRuns[i].ID = key.IntID()
 		fmt.Printf("Added static run for %s\n", staticTestRuns[i].BrowserName)
 	}
+
+	log.Println("Adding static interop data...")
```

Comment:
Note: This is copied from `populate_dev_data.go` so doesn't require much scrutiny.